util/querysets.py

```diff
@@ -0,0 +1,43 @@
+from projects.models import Project
+from allocations.models import Allocation
+from django.db.models import Max, Prefetch
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class CCQuerySets:
+    def allocationsQuerySet(self):
```

Comment:
Style is to do `camel_case`

Reply:
> This looks like a nice change to me! I left some comments. I think the biggest value right now is having pagination just sorted out as a core concern.
> 
> BTW, is it easy to call these paged querysets from other views? Or are they mostly intended for XHR / external API endpoints?

I need to check. The use case appears to be composing views, and rendering based on primary key / hyperlink references between models. I'll add a good example view that exercises the features.